src/components/ToggleSwitch/index.tsx

```diff
@@ -151,7 +140,7 @@ const ToggleSwitch: React.FC<ToggleSwitchProps> = ({
 
   const positionInterPol = positionButton.interpolate({
     inputRange: [0, 1],
-    outputRange: [0, dimen.normalizeDimen(22)]
+    outputRange: [0, dimen.normalizeDimen(20)]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The output range of the `positionInterPol` interpolation has been reduced. Ensure that this change doesn't affect the animation of the toggle switch.

```diff
-    outputRange: [0, dimen.normalizeDimen(22)]
+    outputRange: [0, dimen.normalizeDimen(20)]
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->